README.rst

```diff
@@ -80,3 +80,5 @@ OpenTURNS Presentations
   - `Physical constraints for generative adversial networks (ONERA) <https://github.com/openturns/presentation/blob/master/high-dim-meta-2021/GAN_nuages_OT_ONERA.pdf>`_
   - `High dimension meta-modeling as a preconditioner for aeroacoustic simulation (Airbus) <https://github.com/openturns/presentation/blob/master/high-dim-meta-2021/Metamodelisation_Goldstein_Airbus.pdf>`_
   - `Auto-associative models as a nonlinear PCA for dimension reduction (Phimeca) <https://github.com/openturns/presentation/blob/master/high-dim-meta-2021/modele_autoassociatif_PHIMECA.pdf>`_
+
+- `SIAM-UQ 2022 <https://github.com/openturns/openturns.github.io/blob/master/presentation/master/siamuq-2022-slides-OpenTURNS.pdf>`_
```

Comment:
check the url

Reply:
I don't see what's wrong with the URL: it is exactly the same as for SIAMUQ 2020, except "2020" is replaced by "2022".

Reply:
yes, sorry